ChangeLog.md

```diff
@@ -20,6 +20,11 @@ See docs/process.md for more on how version tagging works.
 
 3.1.75 (in development)
 -----------------------
+- When using `-sMODULARIZE` we now assert if the factory function is called with
+  the JS `new` keyword.  e.g. `var a = new Module()` rather than `var b =
+  Module()`.  This paves the way for marking the function as `async` which does
```

Comment:
The split of backticks over lines looks like it is breaking formatting here, so maybe best to keep `var b = Module()` on one line

Reply:
That seems odd..